app/components/application/AppEndpointUrl/AppEndpointUrl.tsx

```diff
@@ -12,19 +19,57 @@ export const links = () => {
 
 type AppEndpointUrlProp = InputProps & {
   chain: Blockchain | undefined | null
+  handleRemove: () => void
+  hasDelete: boolean
 }
 
-export default function AppEndpointUrl({ chain, ...props }: AppEndpointUrlProp) {
+export default function AppEndpointUrl({
+  chain,
+  handleRemove,
+  hasDelete,
+  ...props
+}: AppEndpointUrlProp) {
+  const theme = useMantineTheme()
+
   if (!chain) {
     return <></>
   }
 
   return (
     <div className="pokt-app-endpoint-url">
-      <div className="pokt-app-endpoint-url-abbrv">
-        <ChainWithImage chain={chain.description} label={chain.ticker} />
-      </div>
-      <TextInput {...props} />
+      <Badge
+        fullWidth
+        color="gray"
+        leftSection={
+          getImageForChain(chain.description || "") ? (
+            <img
+              alt={chain.description || ""}
+              height={16}
+              src={getImageForChain(chain.description || "")}
+            />
+          ) : null
+        }
+        p="12px 0"
+        variant="outline"
+        w={100}
+      >
+        {chain.description?.substring(0, 3).toUpperCase()}
+      </Badge>
+      <TextInput {...props}>
+        <CopyText text={String(props.value)} />
+        {hasDelete && (
+          <Button
+            aria-label="delete"
+            className="pokt-button-outline"
+            color={theme.colors.blue[5]}
```

Comment:
id be interested in seeing what these buttons would look like as "gray". I think removing the outline helped, but the blue still makes them be more visual than I think they need to be.